google-cloud-pubsub/lib/google/cloud/pubsub/subscription.rb

```diff
@@ -693,11 +693,12 @@ def wait_for_messages max: 100
         #
         #   Hash keys and values may include the following:
         #
-        #     * `:limit` (Integer) The number of received messages to be collected by subscriber. Default is 1,000.
-        #     * `:bytesize` (Integer) The total bytesize of received messages to be collected by subscriber. Default is
-        #       100,000,000 (100MB).
-        #     * `:extension` (Integer) The number of seconds that received messages can be held awaiting processing.
-        #       Default is 3,600 (1 hour).
+        #     * `:max_outstanding_messages` (or `:limit`) [Integer] The number of received messages to be collected by
+        #       subscriber. Default is 1,000.
+        #     * `:max_outstanding_bytes` (or `:bytesize`) [Integer] The total byte size of received messages to be
+        #       collected by subscriber. Default is 100,000,000 (100MB).
+        #     * `:max_total_lease_duration` (or `:extension`) [Integer] The number of seconds that received messages can
+        #       be held awaiting processing. Default is 3,600 (1 hour).
```

Comment:
I think we should make it more clear that the old keys are deprecated aliases. For example:
```
 * `:max_outstanding_messages` [Integer] The number of received messages to be collected by
   subscriber. Default is 1,000.
   (Note: `:index` is a deprecated alias for `:max_outstanding_messages`.)
```